fine-mapping/summary_statistics_finemapping.ipynb

```diff
@@ -376,7 +376,7 @@
     "    }\n",
     "    library(susieR)\n",
     "    L = min(${L}, length(z_scores))\n",
-    "    s = susie_z(z_scores, r, r_tol = 5e-3, L = L, min_abs_corr = 0.1,\n",
+    "    s = susie_rss(z_scores, r, r_tol = 5e-3, L = L, min_abs_corr = 0.1,\n",
```

Comment:
@sq-96 `z_ld_weight` is a crucial parameter that hsould be included. You probably have to pass in an additional parameter "ref_panel_size" . Otherwise the result may not be reliable as shown in the warning message of your data. See also this section https://stephenslab.github.io/susieR/articles/finemapping_summary_statistics.html#using-ld-from-reference-panel for before and after using the parameter.